e2e/jail/jail_rpc_test.go

```diff
@@ -88,7 +88,7 @@ func (s *JailRPCTestSuite) TestIsConnected() {
 	response, err := responseValue.ToString()
 	s.NoError(err, "cannot parse result")
 
-	expectedResponse := `{"jsonrpc":"2.0","result":true}`
+	expectedResponse := `{"result":true}`
```

Comment:
Why?

Reply:
According to https://github.com/ethereum/wiki/wiki/JavaScript-API#web3isconnected, `web3.isConnected` should return boolean. I simplified it to `{"result":true}` as `"jsonrpc":"2.0"` is not requied and used. I also added a `TODO` comment to `createIsConnectedHandler()` to remove `result` wrapped completely but it will require some changes in status-react as well.

Reply:
I may be missing some background now but if it's json-rpc, shouldn't `jsonrpc: 2.0` be included in the response?

Reply:
This is not a JSON-RPC, it's a JavaScript API method which uses JSON-RPC to communicate with a server. `web3.isConnected()` sends a JSON-RPC request, waits for JSON-RPC valid response, unwraps a result and returns it.

Reply:
Ah, so it's an already unwrapped response, correct?

Reply:
No, it's wrapped like this:
```js
return {"result": true | false};
```
and it should be just
```js
return true | false;
```
but, as I said, I am not sure it won't break integration with status-react.

Reply:
That's a good point. Let's first clarify with the other team after devcon.